scripts/travis/deploy/publish.sh

```diff
@@ -1,26 +1,27 @@
 #!/usr/bin/env bash
```

Comment:
```suggestion
#!/bin/bash -e
```

always have this in every script so it would abort and exit with error at the first command failure

not using `-e` is the first reason of silently failing green builds

Reply:
Changed